cmd/otel-agent/config/agent_config.go

```diff
@@ -25,6 +25,36 @@ import (
 	"go.opentelemetry.io/collector/service"
 )
 
+type logLevel int
+
+const (
+	off logLevel = iota
+	critical
+	err
+	warn
+	info
+	debug
+	trace
+)
+
+type level struct {
+	value   logLevel
+	mapping string
```

Comment:
Is this field ever used (except when this struct is initialized)?

Reply:
It used to, but we can clean it up, not much value in keeping it around.